packages/engine-core/src/tasks/bootstrap/questions/installTemplate.ts

```diff
@@ -113,9 +112,14 @@ const Question = async (data: NewProjectData) => {
     const npmCacheDir = path.join(c.paths.project.dir, RnvFolderName.dotRnv, RnvFolderName.npmCache);
 
     if (localTemplatePath) {
+        // testing purposes - instant reject
         if (!fsExistsSync(localTemplatePath)) {
             return Promise.reject(`Local template path ${localTemplatePath} does not exist`);
         }
+        const templateConfigPath = path.join(localTemplatePath, RnvFileName.renativeTemplate);
+        if (!fsExistsSync(templateConfigPath)) {
+            return Promise.reject(`Renative template config ${templateConfigPath} does not exist`);
+        }
```

Comment:
I'm following the current reject text format, but maybe make it more informative? I don't think I would instantly understand the problem here if I got this error.